### PR TITLE
switch r-kvstore sdk to r_kvstore

### PR DESCRIPTION
**Pull request by handsomepotato**
image
image

### CODE REVIEW

File: alicloud/data_source_alicloud_kvstore_instance_classes.go [hunk 1]
```diff
@@ -29,41 +29,60 @@ func dataSourceAlicloudKVStoreInstanceClasses() *schema.Resource {
...
 			"engine_version": {
 				Type:     schema.TypeString,
 				Optional: true,
 				ForceNew: true,
 			},
+			"performance_type": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringInSlice([]string{"standard_performance_type", "enhance_performance_type"}, false),
```
Comment (xiaozhu36) [inline alicloud/data_source_alicloud_kvstore_instance_classes.go]: 已经弃用的参数，要加一个Deprecated: "The parameter 'performance_type' has been deprecated from 1.68.0."

File: website/docs/d/kvstore_instance_classes.html.markdown [hunk 2]
```diff
@@ -42,11 +42,14 @@ The following arguments are supported:
 * `engine` - (Optional) Database type. Options are `Redis`, `Memcache`. If no value is specified, all types are returned.
 * `engine_version` - (Optional) Database version required by the user. Value options of Redis can refer to the latest docs [detail info](https://www.alibabacloud.com/help/doc-detail/60873.htm) `EngineVersion`. Value of Memcache should be empty.
 * `architecture` - (Optional) The KVStore instance system architecture required by the user. Valid values: `standard`, `cluster` and `rwsplit`.
-* `performance_type` - (Optional) The KVStore instance performance type required by the user. Valid values: `standard_performance_type` and `enhance_performance_type`.
-* `storage_type` - (Optional) The KVStore instance storage space required by the user. Valid values: `inmemory` and `hybrid`.
+* `performance_type` - (Optional, Deprecated in 1.69.0+) The KVStore instance performance type required by the user. Valid values: `standard_performance_type` and `enhance_performance_type`.
```
Comment (xiaozhu36) [inline website/docs/d/kvstore_instance_classes.html.markdown]: `performance_type` - (Optional, Deprecated) It has been deprecated from 1.68.0.
其他的也一样

File: website/docs/d/kvstore_instance_classes.html.markdown [hunk 3]
```diff
@@ -42,11 +42,14 @@ The following arguments are supported:
...
+* `storage_type` - (Optional, Deprecated in 1.69.0+) The KVStore instance storage space required by the user. Valid values: `inmemory` and `hybrid`.
 * `node_type` - (Optional) The KVStore instance node type required by the user. Valid values: `double`, `single`, `readone`, `readthree` and `readfive`.
-* `package_type` - (Optional) The KVStore instance package type required by the user. Valid values: `standard` and `customized`.
+* `package_type` - (Optional, Deprecated in 1.69.0+) The KVStore instance package type required by the user. Valid values: `standard` and `customized`.
 * `output_file` - (Optional) File name where to save data source results (after running `terraform apply`).
+* `edition_type` - (Optional, Available in 1.69.0+) The KVStore instance edition type required by the user. Valid values: `Community` and `Enterprise`.
```
Comment (xiaozhu36) [inline website/docs/d/kvstore_instance_classes.html.markdown]: 1.69.0 改成1.68.0 吧

File: alicloud/data_source_alicloud_kvstore_instance_classes.go (hunk 1 above)
Comment (handsomepotato) [reply alicloud/data_source_alicloud_kvstore_instance_classes.go]: done

**Pull request merged**